chapter07/qli/ex5.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+while( <> ){
+  if( /(\w)\1/ ){
+    print $_;
+  }
```

Comment:
Seems to have got this wrong like me. The question is to match two non-white space characters, so <code>\S</code> is our friend.